File: Selenium/Project/Project_Java/Activity1.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");
        String title = driver.getTitle();
        // Verify the website title
        if (title.equals("Alchemy Jobs – Job Board Application")) {
            System.out.println("Website title matches: " + title);
        } else {
            System.out.println("Website title does not match: " + title);
        }
        // Close the browser
        driver.quit();
    }

}

File: Selenium/Project/Python_Project/Activity1.py
from selenium import webdriver

# Initialize the WebDriver
driver = webdriver.Chrome()

try:
    # Navigate to the website
    driver.get('https://alchemy.hguy.co/jobs/')
    
    # Get the title of the website
    title = driver.title
    
    # Verify the title
    assert title == "Alchemy Jobs – Job Board Application", "Website title does not match"
    print("Website title verification successful.")

finally:
    # Close the browser
    driver.quit()